src/Tasks/Common/ConflictResolution/PlatformManifestReader.cs

```diff
@@ -28,9 +28,9 @@ public static IEnumerable<ConflictItem> LoadConflictItems(string manifestPath, L
             {
                 for (int lineNumber = 0; !manifestReader.EndOfStream; lineNumber++)
                 {
-                    var line = manifestReader.ReadLine().Trim();
+                    var line = manifestReader.ReadLine()?.Trim();
 
-                    if (line.Length == 0 || line[0] == '#')
+                    if (line is null || line.Length == 0 || line[0] == '#')
```

Comment:
should this instead be `string.IsNullOrWhitespace(line)`?

Reply:
I think I did that at first but changed it to this because `string.IsNullOrWhitespace` is not recognized by the analyzer as a null-check. Therefore, the call to the indexer in the rest of the or-statement would need to be `line![0] == '#'` which is a little jankier. This seemed like a simpler change since it is more explicit to the analyzer.

Reply:
Looking over this, I might actually prefer doing something like this (meaning, explicit null and length check) instead of using the `string` helper methods because it avoids the use of `!`.

Reply:
😢 Analyzer should be smarter